getting_started/cavs-boot/cavs-dsp-boot-overview.rst

```diff
@@ -0,0 +1,126 @@
+
+Overview
+########
+
+There are two main DSP boot flows:
+
+* **Cold boot** performed when the host CPU exits an Sx state. FW binaries are
+  loaded into DSP memory and full state re-initialization is required. This
+  flow is also referred as *Purge Flow* in the figures below.
+
+* **RTD3 boot** when the DSP state is restored from the DSP internal memory.
+  This flow is available on platforms with access to Isolated Memory Region
+  (IMR) allocated for the DSP.
+
+IPC Communication with DSP ROM
+******************************
+
+Once the master DSP core (#0) is powered up and reset by the host driver, an
+IPC communication with the DSP ROM is required in order to set the boot
+options (see Boot Path Control Messages for details and list of platforms that
+require this step). It is a one-way message that does not require a response
+from the DSP.
+
+There may be some specific requirements about the order of the DSP core reset,
+sending IPC message, and the DSP core unstall operations. It is assumed that
+the following order is required unless specified otherwise by Boot Path
+Control Message in case of a specific platform:
+
+1.	Power up and reset the DSP Core 0,
+#.	Send ROM Control IPC,
+#.	Unstall DSP Core 0.
+
+The ROM Control IPC message includes “purge” parameter that should be set to 1
+in case of the cold boot. Otherwise it may be set to 0 after coming out of
+RTD3 to attempt quicker state restore flow. In the latter case, the driver
+just waits for FW Ready notification (no library loading is needed).
+
+The flow is illustrated in the next figure.
+
+.. uml:: images/boot-dsp.pu
+
+Loading Binaries to ADSP Memory
+*******************************
+
+The ADSP FW binary code may be divided into:
+
+* The Base FW binary file, which contains FW infrastructure code (Base FW
+  module) required by all the platforms, optionally followed by other modules,
+
+* Set of libraries (modules) containing additional processing modules code
+  that may be optionally loaded into ADSP FW memory based on the platform’s
+  requirements and configuration.
+
+.. note:: This section contains general information about the structure of
+   binaries necessary to understand the loading process. For a complete
+   documentation refer to FW Binaries documentation.
+
+There are two main parts of the main binary:
+
+*	Manifest,
+*	Modules binary code.
+
+Determining Part of Binary to be Loaded
+---------------------------------------
```

Comment:
We've been following a convention of headings using #, *, =, and -  for specifying heading levels.  It's not critical, but you should be consistent throughout the documents.